tt_metal/hw/inc/dev_msgs.h

```diff
@@ -299,7 +300,8 @@ struct addressable_core_t {
 };
 
 // TODO: This can move into the hal eventually, currently sized for WH.
-constexpr static std::uint32_t MAX_NON_WORKER_CORES = 36 + 1 + 16;
+constexpr static std::uint32_t MAX_NON_WORKER_CORES = 18 + 1 + 16;
+constexpr static std::uint32_t MAX_VIRTUAL_NON_WORKER_CORES = 18;
```

Comment:
```suggestion
constexpr static std::uint32_t MAX_NON_WORKER_CORES = MAX_VIRTUAL_NON_WORKER_CORES + 1 + 16;
```

Also can you add a comment explaining that `MAX_VIRTUAL_NON_WORKER_CORES = 18` because that is the tensix offset in virtualized space and that it is subject to change based on diff archs virtualization scheme

Reply:
I applied the suggested change. This is 18 because we have 18 Ethernet cores on WH. These are the only non-worker cores that are accessible through Virtual Coordinates and its just coincidental that Tensix Coordinates start at 18-18 in the Virtual Space.
I've added a comment clarifying this, as per your suggestions.